.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  'extends': 'eslint:recommended',
+    'parserOptions': {
+      'sourceType': 'module',
+      'ecmaVersion': 2017
+  },
+  'env': {
+    'node': true,
+    'es6': true,
+    'mocha': true,
+    'browser': true
```

Comment:
I'd recommend not to write environment globally. 

You see, this can be an issue in files, which should, let's say, work only in browser env. ESLint will lint such as like they should work in Node too. And that might potentially leak some unexpected error. And current env will treat all files like test files, which imposes same risks. Though, I agree that it's very theoretical thing.

Not related to `es6`, of course.

Reply:
...could be worked into the build toolchain. we're already using webpack to bundle the web library, wouldn't be a huge jump to use it to bundle the server-side code as well. I'm mostly a TS guy, but if I recall correctly, eslint is pretty straightforward to integrate with Webpack.